docs/starting/first_steps.md

```diff
@@ -34,6 +52,27 @@ This works for most other file types that QuPath can handle - not only images.
 A whole slide image (*CMU-1.svs*) opened within QuPath.
 :::
 
+### Setting the image type
+
+One of the most important properties is the **Image type**, which can impact the behavior of some QuPath commands. When opening an image in QuPath you will be presented with a window allowing you to choose the image type. You should choose the closest match for the image, e.g. *Brightfield*, *Fluorescence*.
+
+:::{figure} images/steps_image_type.jpg
+:align: center
+:class: shadow-image
+:width: 75%
+
+Option window for setting the image type in QuPath.
+:::
+
+:::{important}
+QuPath can automatically estimate the image type for you, if you choose this option for *Set image type* in under {menuselection}`Edit --> Preferences...` {{ icon_cog }}.
```

Comment:
I think we should highlight first that the user should press 'Show details'. I think you can use
```
{guilabel}`Show details`
```
to highlight the fact it's a UI button.

You could note that option can always be changed in the preferences, but I think flagging that 'Show details' buttons really are useful could help more.